app/src/test/java/io/appium/uiautomator2/utils/w3c/W3CActionsPreprocessingTests.java

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
any reason to move these tests?

Reply:
Unit tests should be located at ``src/test/java``` folder